src/pytest/test_pair.py

```diff
@@ -17,6 +18,46 @@
 
 potential_tests = []
 
+# pair Hertz
+potential_tests += [
+    PotentialTestCase(
```

Comment:
Can we mix in short comment lines for each of these about why they were chosen, for future reference?

Reply:
I think that's a great idea - it will help future development to see why/what is tested.